src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -301,11 +297,6 @@ class BaseOptionsSelector extends Component {
                 </View>
                 {shouldShowFooter && (
                     <FixedFooter>
-                        {this.props.shouldShowOfflineMessage && this.props.network.isOffline && (
-                            <Text style={[styles.formError, styles.pb2]}>
-                                {this.props.translate('session.offlineMessage')}
```

Comment:
This translation appears unused. We should remove it from the translations files

Reply:
Updated!